service/worker/scanner/history/scavenger_test.go

```diff
@@ -83,14 +86,36 @@ func (s *ScavengerTestSuite) SetupTest() {
 
 func (s *ScavengerTestSuite) createTestScavenger(
 	rps int,
-) (*persistence.MockExecutionManager, *historyservicemock.MockHistoryServiceClient, *Scavenger, *gomock.Controller) {
+) (
```

Comment:
The number of return arguments here makes it hard to use this fn. Also, this function is called at the start of every test function, with the exact same arguments, so I think it should just go to SetupSuite, and then the return args can just be stored in the suite.

Reply:
Will refactor

Reply:
Fixed the test